applications/Chat/examples/train_prompts.py

```diff
@@ -11,9 +11,10 @@
 from coati.trainer import PPOTrainer
 from coati.trainer.strategies import DDPStrategy, GeminiStrategy, LowLevelZeroStrategy
 from torch.optim import Adam
+from torch.optim.lr_scheduler import CosineAnnealingLR
```

Comment:
The naming of this file is a little confusing. I recomment that we name stage-1 as `train_sft.py` and stage-3 as `train_rlhf.py`. What do you think of this?